heffalump/heffalump.go

```diff
@@ -50,8 +58,24 @@ func (h *Heffalump) WriteHell(bw *bufio.Writer) (int64, error) {
 	buf := h.pool.Get().([]byte)
 	defer h.pool.Put(buf)
 
-	if _, err = bw.WriteString("<html>\n<body>\n"); err != nil {
-		return n, err
+	switch cType {
+	case PlainText:
+		if _, err = bw.WriteString("# Chapter 1\n"); err != nil {
```

Comment:
_note: this may actually make my comment about the `break`s irrelevant as well as you wouldn't even need a switch_

more nitpicking: we can probably just define a map, something like

```go
var appleTree = map[ContentType]string{HTML: "<html>\n<body>\n", foo: "bar", } // etc
// [...]
var (
	forbiddenFruit string
	ok bool
)
if forbiddenFruit, ok = appleTree[cType]; !ok {
	panic("this is a tomato") 
}
if n, err = bw.WriteString(forbiddenFruit); err != nil {
	return n, err
}
// [...]
```

something like this is probably a tidier solution, obviously you don't need to be as silly as i'm being here in terms of  var naming

Reply:
> actually i set this up because i needed the markovmap to also escape newlines and quotation marks, but i never got to that part :/

a bit confused as to what you mean, couldn't one just use "`" characters to wrap the string values of the map?

Reply:
im pretty sure that is not valid in JSON

Reply:
the json is _inside_ the backticks, not the other way around

Reply:
no, i mean that everything sent from the markov is escaped, like newlines become literal `\n`s, and quotation marks become `\"`s, or else an iterative parser can stop processing immediately

im not sure how to add a writer that will edit it on the fly

Reply:
i'm clearly missing something, let me take a deeper look at this